pkg/interfaces/contracts/vault/ICompositeLiquidityRouter.sol

```diff
@@ -152,7 +170,7 @@ interface ICompositeLiquidityRouter {
      * @param parentPool Address of the highest level pool (which contains BPTs of other pools)
      * @param tokensIn Input token addresses, sorted by user preference. `tokensIn` array must have all tokens from
      * child pools and all tokens that are not BPTs from the nested pool (parent pool).
-     * @param exactAmountsIn Amount of each underlying token in, sorted according to tokensIn array
+     * @param exactAmountsIn Amount of each token in, sorted according to tokensIn array
```

Comment:
```suggestion
     * @param exactAmountsIn Amount of each token in, corresponding to `tokensIn`
```
This seems more generic, as we might be interpreting tokens differently, such that they're not necessarily sorted